packages/repository/src/__tests__/unit/decorator/relation.decorator.unit.ts

```diff
@@ -198,6 +198,55 @@ describe('relation decorator', () => {
         },
       });
     });
+
+    it('accepts additional property metadata', () => {
+      @model()
+      class AddressBook extends Entity {
+        @property({
+          id: true,
+          length: 36,
```

Comment:
AFAICT, this additional configuration for `AddressBook#id` property is not relevant to this particular test. Please simplify the property definition to provide only the metadata that's needed to make this test fail now and pass with your fix in place.

I think the following should work?

```ts
@model()
class AddressBook extends Entity {
  @property({id: true})
  id: string;
}
```

Reply:
done